db/test.sql

```diff
@@ -10,5 +10,5 @@
   }*/
 
 
-SELECT * FROM articles
-WHERE article_id = 1;
+SELECT * FROM articles;
```

Comment:
try not to push these files to github!

only `git add` the correct files and/or add these files to your `.gitignore`!